packages/repository/test/unit/repository/legacy-juggler-bridge.ts

```diff
@@ -96,6 +96,16 @@ describe('DefaultCrudRepository', () => {
     const notes = await repo.find({where: {title: 't1'}});
     expect(notes.length).to.eql(1);
   });
+  
+  it('implements Repository.findOne()', async () => {
+    const repo = new DefaultCrudRepository(Note, ds);
+    await repo.createAll([
+      {title: 't1', content: 'c1'},
+      {title: 't1', content: 'c2'},
+    ]);
+    const notes = await repo.findOne({where: {title: 't1'}});
+    expect(notes.length).to.eql(1);
```

Comment:
This assertion is wrong. For findOne(), the result is not an array but the instance. Try:
```
const note = await repo.findOne({where: {title: 't1'}, order: 'content DESC'});
expect(notes.title).to.eql('t1');
expect(notes.content).to.eql('c2');
```

After you make the change, run `npm run bootstrap && npm run test` under `loopback-next` to make sure the build is passing.

Reply:
Sorry, forgot to update the test after my last change.